style.css

```diff
@@ -325,3 +324,81 @@ body {
 .lang .lang-exp .no-1 span i {
   color: #acb7c3;
 }
+
+/* contact form */
+.contact-section {
+  background-color: #6070ff;
+  background-image: url(/images/contact-graph-2.png), url(/images/contact-graph-1.png), url(/images/contact-circle-1.png);
+  background-repeat: no-repeat, no-repeat, no-repeat;
+  background-size: 95px, 100px, 28px;
+  background-position: 77% 95%, 110% 3%, 40% 10%;
```

Comment:
- The form background image is not showing up as intended, please, kindly double-check to ensure you've fixed this issue.

Reply:
### README FILE


- Please, kindly update your readme to follow the current readme [template](https://github.com/microverseinc/readme-template) structure and format to improve and make your readme more professional.

Reply:
Fixed.